src/core/libmaven/mavenparameters.cpp

```diff
@@ -465,6 +467,28 @@ bool MavenParameters::loadSettings(const char* data)
     return true;
 }
 
+
+void MavenParameters::reset(const std::list<string>& keys)
```

Comment:
@rish9511 vector will be much faster than list but this is also fine as we have few keys. Pointing out for future usage.

Reply:
@sahil21 ya, we have hardly 20-30 keys. so won't make a difference

Reply:
@rish9511 Cool